packages/client/src/components/Location/LocationFinder.tsx

```diff
@@ -34,7 +34,7 @@ const LocationFinder: React.FC<{
   setAddress: Function;
   setErrorMessage: Function;
   setFocus: Function;
-  topic: any; //@TODO: Replace it with IMTR-Client's TopicType
+  topic: Topic; // TODO: Remove topic from this component.
```

Comment:
This is still sort of todo right now

Reply:
Yes, now at least it's typed. In react-hooks-poc I'll remove it completely.